object/organization.go

```diff
@@ -46,7 +46,7 @@ type Organization struct {
 	EnableSoftDeletion bool     `json:"enableSoftDeletion"`
 	IsProfilePublic    bool     `json:"isProfilePublic"`
 
-	AccountItems []*AccountItem `xorm:"varchar(2000)" json:"accountItems"`
+	AccountItems []*AccountItem `xorm:"varchar(3000)" json:"accountItems"`
```

Comment:
why change this?

Reply:
Because when I store too many of AccountItems, the database will report an out-of-memory error.

Reply:
OK